Pyrado/pyrado/logger/experiment.py

```diff
@@ -132,10 +143,16 @@ def matches(self, hint: str) -> bool:
             return self.env_name == env_name and self.algo_name == algo_name and self.exp_id == eid
 
 
-def setup_experiment(env_name: str, algo_name: str, extra_info: str = None, base_dir: str = pyrado.TEMP_DIR):
+def setup_experiment(
+    env_name: str,
+    algo_name: str,
+    extra_info: str = None,
+    base_dir: str = pyrado.TEMP_DIR,
+    include_slurm_id: bool = True,
+):
     """ Setup a new experiment for recording. """
```

Comment:
the docsting must explain every arument (:param: ... and optionally :return: ...)